keywords.txt

```diff
@@ -18,9 +18,16 @@ end	KEYWORD2
 readTemperature	KEYWORD2
 readHumidity	KEYWORD2
 
+enableDataReady KEYWORD2
+disableDataReady    KEYWORD2
+setOpenDrain    KEYWORD2
+setPushPull KEYWORD2
+setActiveHigh   KEYWORD2
+setActiveLow    KEYWORD2
```

Comment:
```suggestion
enableDataReady	KEYWORD2
disableDataReady	KEYWORD2
setOpenDrain	KEYWORD2
setPushPull	KEYWORD2
setActiveHigh	KEYWORD2
setActiveLow	KEYWORD2
```
In order to be recognized by the Arduino IDE, the keyword fields must be separated by a single true tab (not spaces):
https://arduino.github.io/arduino-cli/latest/library-specification/#keywordstxt-format